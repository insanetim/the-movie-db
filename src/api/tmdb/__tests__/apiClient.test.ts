import tmdbClient from '../apiClient'

describe('tmdbClient', () => {
  it('should match snapshot', () => {
    expect(tmdbClient).toMatchSnapshot()
  })
})
