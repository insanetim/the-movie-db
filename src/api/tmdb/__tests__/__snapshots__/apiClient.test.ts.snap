// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmdbClient should match snapshot 1`] = `[Function]`;
